common/faceting.js

```diff
@@ -685,19 +690,87 @@
                         }
                     }
 
+                    var FLOAT_PRECISION = 10000;
+                    function initializeRangeMinMax(min, max) {
```

Comment:
I wasn't sure exactly how much to document but I think I definitely need a jsdoc like comment explaining this function since there is a reason it's being used instead of `setRangeMinMax`.